src/bindist/fdppauto.bat

```diff
@@ -22,3 +22,4 @@ if not "%DOSEMU_EXIT%" == "1" goto noexit
 exitemu %ERRORLEVEL%
 :noexit
 if exist %USERDRV%:\userhook.bat %USERDRV%:\userhook.bat
+if exist %DOSEMUDRV%:\dosemu\insfdusr.bat if not exist %USERDRV%:\bin\*.exe echo FreeDOS userspace tools can be installed with: insfdusr
```

Comment:
Maybe something lile
`if exist %DOSEMUDRV%:\dosrc.d\fdsuggest.bat call %DOSEMUDRV%:\dosrc.d\fdsuggest.bat`?
Would be easier to translate to hooks later.

Reply:
`fdsuggest.bat` does not conform to the 8.3 filename format. Do you think that limitation could be broken or should I use a shorter name?